### ISSUE TITLE
Update CompatibilityPolicy and link to it prominently

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3263]
|[<img alt="jml's avatar" src="https://avatars.githubusercontent.com/u/125674?s=50" width="50" height="50">](https://github.com/jml)| @jml reported|
|-|-|
|Trac ID|trac#3263|
|Type|enhancement|
|Created|2008-06-01 04:26:01Z|

CompatibilityPolicy needs to updated to match the recent mailing list thread starting at http://twistedmatrix.com/pipermail/twisted-python/2008-April/017497.html

We need to link to the compatibility policy prominently. I suggest:
 * The FAQ
 * The coding standards

<details><summary>Searchable metadata</summary>

```
trac-id__3263 3263
type__enhancement enhancement
reporter__jml jml
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__duplicate duplicate
component__website website
keywords__deprecation_policy_documentation deprecation-policy documentation
time__1212294361000000 1212294361000000
changetime__1461652291499000 1461652291499000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra commented|
|-|-|

Replying to [jml](#3263):
> We need to link to the compatibility policy prominently. I suggest:
>  * The FAQ

This should be fixed, see [FrequentlyAskedQuestions#WhichpartsofTwistedarestable](https://twistedmatrix.com/trac/wiki/FrequentlyAskedQuestions#WhichpartsofTwistedarestable)

>  * The coding standards
>

I'll add a link to the coding-standard.

**Comment by twisted-trac** [comment]
|<img alt="binjured's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|binjured commented|
|-|-|

Sooo... this is done?

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_6"></a>|@adiroiban set status to `closed`|
|-|-|

I think this was fixed by [#8082](https://github.com/twisted/twisted/issues/8082)
